### DISCUSSION TITLE
Make left arrow back to input

### DISCUSSION THREAD

**Discussion by MadeBaruna** [marlonrichert/zsh-autocomplete#655] [Q&A] [answered]
Just updated the plugin, as I remembered, the left arrow will select the history and I can edit it immediately, how to get back to the old behavior? Tried to add `bindkey -M menuselect "^[[D" backward-char` but it doesn't work.

**Comment by JerryZhongJ** [comment]
I borrowed the idea from README.md, and this one work for me:
```
() {
   local -a prefix=( '\e'{\[,O} )
   local -a left=( ${^prefix}D ) right=( ${^prefix}C )
   local key=
   for key in $left[@]; do
      bindkey -M menuselect "$key" .backward-char
   done
   for key in $right[@]; do
      bindkey -M menuselect "$key" .forward-char
   done
}
```

**Comment by marlonrichert** [comment, marked as answer]
It's documented in the Readme: [Make ← and → always move the cursor on the command line](Make ← and → always move the cursor on the command line)
